rio_tiler/io/base.py

```diff
@@ -806,7 +825,11 @@ def _reader(band: str, **kwargs: Any) -> Info:
 
         bands_metadata = multi_values(bands, _reader, *args, **kwargs)
 
-        meta = self.spatial_info.dict()
+        meta = {
+            "bounds": self.bounds,
+            "minzoom": self.minzoom,
+            "maxzoom": self.maxzoom,
+        }
```

Comment:
I've decided to remote the `spatial_info` method because it was kinda useless. I also removed `center` property.